src/extension.ts

```diff
@@ -346,14 +346,20 @@ function startWebhint(context: vscode.ExtensionContext): void {
     void client.onReady().then(() => {
         // Listen for notification that the webhint install failed.
         const installFailedNotification: typeof installFailed = 'vscode-webhint/install-failed';
-        client.onNotification(installFailedNotification, () => {
-            const message = 'Ensure `node` and `npm` are installed to enable automatically reporting issues in source files pertaining to accessibility, compatibility, security, and more.';
-            void vscode.window.showInformationMessage(message, 'OK', 'Disable').then(button => {
-                if (button === 'Disable') {
-                    void vscode.workspace.getConfiguration(SETTINGS_STORE_NAME).update('webhint', false, vscode.ConfigurationTarget.Global);
-                }
+        const disableNotication = vscode.workspace.getConfiguration(SETTINGS_STORE_NAME).get('webhintInstallNotification');
+        if (!disableNotication) {
+            client.onNotification(installFailedNotification, () => {
+                const message = 'Ensure `node` and `npm` are installed to enable automatically reporting issues in source files pertaining to accessibility, compatibility, security, and more.';
+                void vscode.window.showInformationMessage(message, 'Remind me Later', 'Disable Extension', 'Don\'t show again').then(button => {
+                    if (button === 'Disable Extension') {
+                        void vscode.workspace.getConfiguration(SETTINGS_STORE_NAME).update('webhint', false, vscode.ConfigurationTarget.Global);
+                    }
+                    if (button === 'Don\'t show again') {
```

Comment:
WDYT about moving `Disable Extension` at the end. My mental model is that it should follow (left to right ): (Disable this for now), (Disable this always), (Disable ALL always)